sources/packages/web/src/components/common/Reports.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <full-page-container>
+    <template #header>
+      <header-navigator :title="title" subTitle="Reports" />
+    </template>
+    <body-header title="Export reports" />
+    <formio
+      formName="exportfinancialreports"
+      @loaded="formLoaded"
+      @submitted="exportReport"
+    ></formio>
+    <v-row class="justify-center m-4">
+      <check-permission-role :role="Role.AESTReports">
+        <template #="{ notAllowed }">
+          <v-btn color="primary" @click="submitForm" :disabled="notAllowed"
```

Comment:
Can we take the opportunity to add a simple loading to the button?
Righ now there is no feedback at all to the user and the report loading is a long operation.